Aztec/Classes/TextKit/TextView.swift

```diff
@@ -611,11 +613,13 @@ open class TextView: UITextView {
         formatter.toggleAttribute(inTextView: self, atRange: range)
     }
 
-    /// Refresh Lists attributes when text is deleted in the specified range
+
+    /// Toggles the Blockquote Style, whenever the deleted text is at the Location 0, which is 
+    /// also known as the "Beginning of the text"
```

Comment:
I think it'd be better if we bring back the original description, and concatenate your new documentation below it as a more detailed description of what happens.

The reason being that the original description described what the external usage for this class is, and when external actors would want to use this method, whereas the new documentation explains internal details about what it really does.